.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ test_steps: &test_steps
   - checkout
   - run: echo "deb http://deb.debian.org/debian testing main" >> /etc/apt/sources.list
   - run: apt-get update
-  - run: apt-get install -y xvfb libgirepository1.0-dev libgtk-3-dev # xvfb for running headless tests (headless X11 server).
+  - run: apt-get install -y xvfb libgirepository1.0-dev libgtk-3-dev libkeybinder-3.0-0 gir1.2-keybinder xdotool # xvfb for running headless tests (headless X11 server).
```

Comment:
A bit of a churn with additional dependencies, but I think global hotkeys is essential for desktop apps

Reply:
I agree keybindings are important for desktop applications and i'm happy to keep the example code in the application as a demo for people to run.

I don't think we should include this dependency in the test suite though as it adds to the local environment setup for people looking to contribute. `node-gir` runs on MacOS and should only depend on `gobject-introspection` and `nodejs`. One could argue we shouldn't have included GTK as a testing dependency but I'm not familiar enough with core glib/gobject-introspection to write all the tests against just what's available in those libraries; and so GTK was an obvious choice as it's available on all platforms (including MacOS via `brew install gtk+3`) and ships in most linux distros by default.

So i'd prefer not to add this dependency, but I definitely want to keep the eample app.

Reply:
I see the following resolution: move the test file to a different directory (e.g. `__tests__/integration`), tweak jest config to ignore it by default, add a separate npm script `test:integration` which runs only that directory and a separate CircleCI build with additional dependencies.

What do you think?

Reply:
not a bad idea. Would be good to create seperate setups for linux/mac/windows integration tests and document what people need to install to run them. not sure how many we'll write though as our existing unittests are basically integration tests (testing bindings from JS). we're more likely to start writing c++ tests to unittest core parts of the bindings codebase. I'm happy to accept the proposed change though if you want more than an addition to `examples/`